identifiers/country-gb/constituencies.csv

```diff
@@ -0,0 +1,650 @@
+ocd-division/country:gb/country:gb-eng/region:ukc/cd:berwick-upon-tweed,Berwick-upon-Tweed
```

Comment:
My only comment for this file is considered ed instead of cd for electoral district.

Reply:
In this case (i think) we dont have "electoral districts".
https://en.wikipedia.org/wiki/United_Kingdom_Parliament_constituencies
These are all just normal constituencies.

If we should use also ed for constituencies, then is country-de also wrong.
But please correct me if i'm wrong. Maybe I need a new clear definition of the differences between "ed" and "cd"? ;)

Reply:
`cd` originally meant "congressional district" (for the US). However, it has been used as a more generic 'national election' district type ID, but the question of whether to continue that practice is the subject of #170 – given that many countries do not have congresses. (Similarly, `state` was used for US states, and then re-used for ADM1 divisions, though many countries do not have states.)

`ed` means "electoral district". I'm less fussed about the generic use of `ed` than `cd`, because, even if a division is locally labelled as a "constituency", "riding", "division", "ward", etc., as long as an election occurs that relates to that division, then I think it's okay (semantically) to label it as `ed`.

Reply:
Done